ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/activityServiceAgent.service.ts

```diff
@@ -48,6 +50,9 @@ export class ActivityServiceAgent extends UserServiceAgent<any> {
                 x.activity == null || x.activity === '') {
                 return of(null);
             }
+            this.studyGuid = x.study;
```

Comment:
hey @MocanaAtBroad, not sure if this would be ideal, I'm assigning the value to a global variable in the service in order to be able to get it and send it from the component, what do you think?

Reply:
Technically, we can have an activity as a child of another activity. In this case it can leads to a mess with activityGuid.

We usually pass `studyGuid`/`activityGuid` from an activity level down to a block/question.
E.g.  the closest place: `ddp-sdk/src/lib/components/activityForm/answers/activity-answer/activityAnswer.component.ts`.
Please pay attention at how we provide them as input properties for `<ddp-activity-file-answer>`.